docs/glossary.rst

```diff
@@ -7,58 +7,73 @@ Glossary
    :sorted:
 
    ACM
-      An Access Control Matrix defines the rights of a list of
-      principals. An ACM crosses principals with permissions. At the
-      intersection of a principal and a permission there is an action.
-      The action can be either :data:`pyramid.security.Allow`,
-      :data:`pyramid.security.Deny` or :const:`None`. None is a default value and
-      does not grant any right.
+      An :abbr:`ACM (access control matrix)` maps :term:`permissions
+      <permission>` to :term:`roles <role>`.
 
-   post_pool
-      A normal or :term:`service` :class:`adhocracy_core.interfaces.IPool` that
-      serves as the common place to post resources of a special type for a
-      given context.
-      If `resource sheet` field with backreferences sets a
-      :class:`adhocracy_core.schema.PostPool` field, the
-      referencing resources can only be postet at the :term:`post_pool`.
-      This assumes that a post_pool exists in the :term:`lineage` of the referenced
-      resources.
-      If a `resource sheet` field with references sets this, the
-      referenced resource type can only be posted to :term:`post_pool`.
+   resource
+      Anything that exists in adhocracy is a resource: A proposal, a
+      comment, but also users or individual rates.
 
-   service
-      A resource marked as `service`. Services
-      may provide special rest api end points
-      and helper methods. You can find them by their name with
-      :func:`adhocracy_core.interfaces.IPool.find_service`.
-      The `service` has to be in :term:`lineage` or a child of a
-      :term:`lineage` pool for a given :term:`context`.
+   resource type
+      Think of a resource type as a blueprint, and a :term:`resource` as
+      the actual building you build by following the blueprint. Example:
+      The proposal "Better food in the cafeteria" would have the type
+      ``adhocracy_core.resources.proposal.IProposal``.  Note that all
+      resources of the same type have the same :term:`sheets <sheet>`,
+      so there might be a lot of similar types with slightly different
+      sheets (e.g. a simple proposal, a proposal with a budget, a
+      proposal with a geographical location, ...).
 
-   principal
-       A principal is a string representing a :term:`userid`, :term:`groupid`,
-       or :term:`roleid`. It is provided by an :term:`authentication policy`.
-       For more information about the permission system read
-       :doc:`api/authentication_api`.
+   sheet
+      Sheets are the features of :term:`resources <resource>`. A
+      proposal may for example have the sheet
+      ``adhocracy_core.sheets.title.ITitle`` that allows it to have a
+      title and the sheet ``adhocracy_core.sheets.comment.ICommentable``
+      that allows it to be commented on. A resource is really not much
+      more than the sum of its sheets.
```

Comment:
A resource is the sum of its sheets and the path.
The only general field "Content_type" would fit better in the  metadata sheet.
Apart from this we have different python APIs for Pools and Simples, but this is a drawback due to technical reasons.